src/test/java/FakeUI.java

```diff
@@ -0,0 +1,33 @@
+import java.util.ArrayList;
+
+class FakeUI extends UI {
```

Comment:
I didn't even realize there was a duplicate!

All `FakeUI`s should derive from the same class now (12d4cef)